Recrutify/Recrutify.DataAccess/Repositories/Abstract/IUserRepository.cs

```diff
@@ -10,5 +11,7 @@ public interface IUserRepository : IBaseRepository<User>
         Task<User> GetByEmailAsync(string email);
 
         Task<Dictionary<Guid, string>> GetNamesByIdsAsync(IEnumerable<Guid> ids);
+
+        Task<IEnumerable<User>> GetByRoles(List<Role> roles);
```

Comment:
pass IEnumerable<Role> to be in one style

Reply:
+